modules/rest-util/test/blaze/middleware/fhir/resource_test.clj

```diff
@@ -15,47 +16,75 @@
    [taoensso.timbre :as log])
   (:import
    [java.io ByteArrayInputStream]
-   [java.nio.charset StandardCharsets]))
+   [java.nio.charset StandardCharsets]
+   [java.util Base64$Encoder]
+   [java.util Base64]))
 
 (set! *warn-on-reflection* true)
 (st/instrument)
 (log/set-min-level! :trace)
 
 (test/use-fixtures :each tu/fixture)
 
-(defn wrap-error [handler]
+(defn- wrap-error [handler]
   (fn [request]
     (-> (handler request)
         (ac/exceptionally handler-util/error-response))))
 
-(def resource-handler
-  "A handler which just returns the :body from the request."
+(def ^:private resource-handler
+  "A handler which just returns the :body from a non-binary resource request."
   (-> (comp ac/completed-future :body)
       wrap-resource
       wrap-error))
 
-(defn input-stream
+(def ^:private binary-resource-handler
+  "A handler which just returns the :body from a binary resource request."
+  (-> (comp ac/completed-future :body)
+      wrap-binary-data
+      wrap-error))
+
+(defn- string-input-stream
   ([^String s]
    (ByteArrayInputStream. (.getBytes s StandardCharsets/UTF_8)))
   ([^String s closed?]
    (proxy [ByteArrayInputStream] [(.getBytes s StandardCharsets/UTF_8)]
      (close []
        (reset! closed? true)))))
 
+(defn- binary-input-stream
+  ([^bytes data]
```

Comment:
as I see it, the form without closed? isn;t used - right?

Reply:
Actually, it is. (In the JSON and the XML tests.)
Do you want me to change those tests to use the form with `closed?`?
I could do that in a new PR...

Reply:
I don't wee the usage. Please point me to the line.